src/app/components/application/review-registration/review-registration.component.ts

```diff
@@ -38,5 +38,7 @@ export class ReviewRegistrationComponent implements OnInit {
     this.appDataService.stateData.asObservable().subscribe(states => {
       this.stateCodes = states;
     });
+
+    this.appDataService.reviewCompleted.next(true);
```

Comment:
I'm not convinced the Review page needs a checkbox... It only has one on the Camp/Academy because there's something to do (the & Submit part).

Feels like this would just be confusing to force people to be like "Ohh... that isn't checked, I'll start there" and suddenly it just becomes checked.

Reply:
They want one, this fills the request